eel_hole/templates/partials/search_results.html

```diff
@@ -1,8 +1,19 @@
 {% for r in resources %}
 <div class="block">
   <div class="level">
-    <h2 class="title is-4 level-left">{{r.name}}</h2>
+    <div class="level-left">
+      <span class="title is-4 level-item">{{r.name}}</h2>
+    </div>
   </div>
+  <div class="level">
+    <div class="level-left">
+      {% for source in r.sources %}
+        <span class="tag level-item">database: {{ source["title"] }}</span>
```

Comment:
To reduce cognitive load, it's a good idea to use the same vocabulary in the UI label as in the query syntax -- so either use "database" both places, or use "package" both places, or (if you must) "source" both places.